core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -183,6 +185,8 @@ object ZkUtils {
 /**
  * Legacy class for interacting with ZooKeeper. Whenever possible, ``KafkaZkClient`` should be used instead.
  */
+@deprecated("This is an internal class that is no longer used by Kafka and will be removed in a future release. Please " +
+  "use org.apache.kafka.clients.admin.AdminClient instead.", since = "2.0.0")
```

Comment:
use  KafkaZkClient?

Reply:
No, `KafkaZkClient` is an internal class too. People should use the `AdminClient` for admin functionality which is the common reason for external users to use this class. Does that clarify it?

Reply:
Yes, makes sense.  Thanks for the clarification.